### ISSUE TITLE
I trained a lora embeding with latest note book,but can't load it in auto1111 additonal network extension.

### ISSUE THREAD

**Issue by eleven-monkey** [Linaqruf/kohya-trainer#46]
it give the error information below:
Traceback (most recent call last):
  File "/content/stable-diffusion-webui/modules/scripts.py", line 338, in process
    script.process(p, *script_args)
  File "/content/stable-diffusion-webui/extensions/sd-webui-additional-networks/scripts/additional_networks.py", line 177, in process
    network, info = lora_compvis.create_network_and_apply_compvis(du_state_dict, weight, text_encoder, unet)
  File "/content/stable-diffusion-webui/extensions/sd-webui-additional-networks/scripts/lora_compvis.py", line 73, in create_network_and_apply_compvis
    network_dim = min([s for s in size if s > 1])
ValueError: min() arg is an empty sequence

strangly,the embeding I trained hours before this one works fine.I assume that the lates update may be introduce in bugs.
I trained the embeding with a smaller network dimension：32，the bug seems something related to network dimension.

**Comment by DarkAlchy** [comment]
After 2h of training to then attempt to use it in Automatic1111 I received screen gibberish so I went to bed in utter disgust to wake up and try it again to get the same stuff so I then looked at the CLI to see - Failed to match keys when loading Lora and I get visual gibberish with this.  Did this ever work?  The webgui version is fantastic, and simple to use, but is not on colab, and he is not interested in doing a colab version.
